curvefs/conf/tools.conf

```diff
@@ -35,6 +35,7 @@ s3.bucket_name=bucket
 s3.blocksize=4194304
 s3.chunksize=67108864
 s3.useVirtualAddressing=false
+s3.objectPrefix=0
```

Comment:
It may be more appropriate to use `objectPrefixVersion` instead of `objectPrefix`.
Please add a comment to explain the difference between different versions.

In addition, it is recommended to merge 6 commits into 1 for submission
<img width="798" alt="image" src="https://user-images.githubusercontent.com/15689619/232948262-6f81fb69-b29a-4edf-88a5-024d5c8d8a87.png">

Reply:
ok

Reply:
I think `objectPrefixVersion ` can't express the meaning. 0 means no prefix, And 1/2/3/... means different kinds of prefix.

Reply:
<img width="583" alt="image" src="https://user-images.githubusercontent.com/15689619/232979230-52877171-747f-4942-9d98-8c1c61f393f8.png">

It's not that the prefix is ​​different, but the organizational form of the objectname has changed. Or `objectNameVersion`?

`objectNameVersion=0` means fsid_inodeid_chunkid_index_compaction
`objectNameVersion=1` means fsid_inodeid/1000/1000_inodeid/1000_fsid_inodeid_chunkid_index_compaction

Reply:
There's no meaning, just adds prefix to improve write and sync performance.